webpack.config.js

```diff
@@ -177,6 +177,7 @@ var appConfig = {
       'window.jQuery': 'jquery',
       'root.jQuery': 'jquery',
       Raven: 'raven-js',
+      sdk: 'sdk',
```

Comment:
Is this necessary if we're always just importing from `app/utils/sdk`?

Reply:
Good catch